### PR TITLE
Add 'Quit to desktop' button in pause menu

### PR DESCRIPTION
**Pull request by VladWinner**

### CODE REVIEW

**Comment by JariKCoding** [comment]
Did you see if there is a localized string you could use instead of doing all these if statements?

**Comment by VladWinner** [comment]
There is `MENU_QUIT_CAPS` ('Quit' button from the main menu), while `MENU_QUIT_TO_DESKTOP` ('Quit to desktop' button) was only seen in MW2CR/H2 (2020) and later CoDs

File: data/ui_scripts/party/__init__.lua [hunk 1]
```diff
@@ -80,5 +80,28 @@ DataSources.StartMenuGameOptions = ListHelper_SetupDataSource("StartMenuGameOpti
...
+	elseif CoD.LANGUAGE_POLISH == Dvar.loc_language:get() then
+		table.insert(options, {models = {displayText = "WYJDŹ DO PULPITU", action = OpenPCQuit}})
+	elseif CoD.LANGUAGE_JAPANESE == Dvar.loc_language:get() or CoD.LANGUAGE_FULLJAPANESE == Dvar.loc_language:get() then
+		table.insert(options, {models = {displayText = "デスクトップに戻る", action = OpenPCQuit}})
+	elseif CoD.LANGUAGE_PORTUGUESE == Dvar.loc_language:get() then
+		table.insert(options, {models = {displayText = "SAIR PARA A ÁREA DE TRABALHO", action = OpenPCQuit}})
+	elseif CoD.LANGUAGE_SIMPLIFIEDCHINESE == Dvar.loc_language:get() then
+		table.insert(options, {models = {displayText = "退至桌面", action = OpenPCQuit}})
+	elseif CoD.LANGUAGE_TRADITIONALCHINESE == Dvar.loc_language:get() then
+		table.insert(options, {models = {displayText = "返回桌面", action = OpenPCQuit}})
```
Comment (momo5502) [inline data/ui_scripts/party/__init__.lua:102]: I appreciate the effort, but honestly, fuck localizations. My opinion is: english only.

Comment (VladWinner) [reply data/ui_scripts/party/__init__.lua]: OK

**Review by JariKCoding — APPROVED** [review]
Looks good to me

**Pull request merged**